src/ServicePulse.Host/app/js/dashboard/spMoment.ts

```diff
@@ -0,0 +1,27 @@
+import moment from 'moment';
+
+export default class EventLogItemComponent extends HTMLElement {
+    date: moment.Moment | null = null;
+
+    constructor() {
+        super();
+
+        // Wait for slots to be populated
+        let observer = new MutationObserver((changes, observer) => {
+            observer.disconnect();
+
+            this.date = moment(this.firstChild?.firstChild?.nodeValue);
+            this.innerText = this.date.fromNow();
+
+            this.startUpdateLoop();
+        });
+
+        observer.observe(this, { childList: true});
+    }
+
+    startUpdateLoop() {
+        setInterval(() => {
```

Comment:
Don't need to cancel if this is an MPA as a full reload on navigate will clear this anyway